spec/services/reports/merits_report_creator_spec.rb

```diff
@@ -18,7 +20,6 @@
 
   describe '.call' do
     it 'attaches merits_report.pdf to the application' do
-      expect_any_instance_of(CCMS::Requestors::ReferenceDataRequestor).to receive(:call)
       expect(Providers::MeritsReportsController.renderer).to receive(:render).and_call_original
```

Comment:
I removed the `expect_any_instance_of(CCMS::Requestors::ReferenceDataRequestor).to receive(:call)`
which i believe is the section where the ccms is being called to create a case_reference which is needed in the reports `#create_ccms_submission`. Not only did it fail, I thought having that much knowledge of another class in this test wasn't needed. 

I would appreciate your thoughts on this. If you'd like to keep this, would you mind pairing with me on this please and thank you?